doc/ctrlsf.txt

```diff
@@ -103,6 +103,9 @@ mode:
   5. If you change your mind later, you can always undo it by pressing
   |u| and saving result buffer again.
 
+Note: The result set will additionally be stored in a |location-list| of the
```

Comment:
A bit strange to add this phrase here, moving to the foot of `2.1` looks better.